semantics/Dafny/oldapproachhedy.dfy

```diff
@@ -0,0 +1,306 @@
+// Level 1

```

Comment:
@juliabolt Do we still need this file? If so, might be good to add a bit of text explaining the value of keeping the old approach?

Reply:
I think it might be useful if someone wants to continue with my work, I've added a little info in the readme.